tests/integration/handling-api-messages.js

```diff
@@ -0,0 +1,20 @@
+describe("handling api messages", function () {
+
+  it("handles and re-hydrates commands", function () {
```

Comment:
`sends a handled command on the server-side command bus if passes validation`
`does not send a handled command on the server-side command bus if fails validation`